MODULE.bazel

```diff
@@ -214,6 +214,163 @@ oci.pull(
 )
 use_repo(oci, "alpine_openssl", "alpine_openssl_linux_amd64")
 
+# Haskell toolchain for spec_compliance tests
+
+bazel_dep(
+    name = "rules_haskell",
+    version = "1.0",
+)
+
+# patched to work around https://github.com/tweag/rules_haskell/issues/2254
+archive_override(
+    module_name = "rules_haskell",
+    integrity = "sha384-bL1ZOn6DZpfsl4+a3PA9T4OAOJnioHXUhoRR9jChk4oy9ykbMAjCBpZ+AxRdRu7V",
+    patch_strip = 1,
+    patches = ["//bazel:rules_haskell.patch"],
+    strip_prefix = "rules_haskell-1.0",
+    urls = ["https://github.com/tweag/rules_haskell/releases/download/v1.0/rules_haskell-1.0.tar.gz"],
+)
+
+haskell_toolchains = use_extension(
+    "@rules_haskell//extensions:haskell_toolchains.bzl",
+    "haskell_toolchains",
+)
+haskell_toolchains.bindists(version = "9.6.6")
+
+rules_haskell_dependencies = use_extension(
+    "@rules_haskell//extensions:rules_haskell_dependencies.bzl",
+    "rules_haskell_dependencies",
+)
+use_repo(
+    rules_haskell_dependencies,
+    "Cabal",
+)
+
+# Stack snapshot. To update, run
+# bazel run @stackage-unpinned//:pin
+stack_snapshot = use_extension(
+    "@rules_haskell//extensions:stack_snapshot.bzl",
+    "stack_snapshot",
+)
+use_repo(
+    stack_snapshot,
+    "stackage",
+    "stackage-unpinned",
+)
+stack_snapshot.stack_snapshot_json(label = "//:stackage_snapshot.json")
+stack_snapshot.snapshot(name = "lts-22.30")
+
+[
+    stack_snapshot.package(
+        name = pkg,  # See https://github.com/tweag/rules_haskell/issues/1871
```

Comment:
I looked at that issue but couldn't figure out the applicability to the code here. What's the relevance?

Reply:
it didn't make sense to me but I assumed I had been out of the loop for too long. I copied this comment from the [rules_haskell MODULE](https://github.com/tweag/rules_haskell/blob/bfc80f4d171dd4679e1a3b9e95bf501093af34c3/MODULE.bazel#L322) and assumed it would make sense to _True Haskellers™_

Reply:
I think they're [referring to their custom Cabal here](https://github.com/tweag/rules_haskell/blob/bfc80f4d171dd4679e1a3b9e95bf501093af34c3/MODULE.bazel#L325). I think you can just remove that comment since we're not using a custom Cabal AFAIK.